src/main/java/core/basesyntax/BallColors.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public enum BallColors {

```

Comment:
```suggestion
public enum Color {

```

as you can use this enum not only for balls